src/authorization/authorization.ts

```diff
@@ -767,6 +769,20 @@ export function initialize(
         /* stop adding JWT to requests */
         baseAxiosRequest.interceptors.request.eject(userInterceptor);
       }
+    },
+    refreshJwtToken: async (authType: string) => {
+      /* this will just clear the existing timeout */
+      handleTokenExpiration('');
+      const payloadToPass = validateAuthType(authType)
+        ? { email: authType }
+        : { userID: authType };
+
```

Comment:
this can be simplified a tad: 
```suggestion
      const payloadToPass = {
        [validateAuthType(authType) ? 'email' : 'userID']: authType
      }

```